src/lib/Libwin/accesinfo.c

```diff
@@ -251,16 +268,14 @@ secure_file(char *path, char *user, ACCESS_MASK mask)
 
 	pdacl = create_secure_dacl(user, mask, usid);
 	if (pdacl == NULL) {
-		sprintf(logb, "failed to create secure dacl for file %s", path);
-		log_err(-1, "secure_file", logb);
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "failed to create secure dacl for file %s", path);
```

Comment:
no error code from previous call?

Reply:
Logging of error code already handles inside that API